src/ui/Modal/index.tsx

```diff
@@ -141,24 +141,20 @@ export function Modal(props: ModalProps): ReactElement {
             />
           ))
         }
-        {
-          !isMobile && (
-            <div className="sendbird-modal__close">
-              <IconButton
-                width="32px"
-                height="32px"
-                onClick={handleClose}
-              >
-                <Icon
-                  type={IconTypes.CLOSE}
-                  fillColor={IconColors.DEFAULT}
-                  width="24px"
-                  height="24px"
-                />
-              </IconButton>
-            </div>
-          )
-        }
+        <div className="sendbird-modal__close">
+          <IconButton
+            width="32px"
+            height="32px"
+            onClick={handleClose}
+          >
+            <Icon
+              type={IconTypes.CLOSE}
+              fillColor={IconColors.DEFAULT}
+              width="24px"
+              height="24px"
+            />
+          </IconButton>
+        </div>
```

Comment:
Q1. why isn't it included in header?
Q2. If there is no button available and customer were implementing custom buttons through renderHeader, what would happen?

Reply:
A1. It seems the ModalHeader only contains the Label(text)
A2. hm... I see,, it will be good to move the CloseButton into the ModalHeader! Then the customers' customized header won't break

Let me apply that!

Reply:
![image](https://github.com/sendbird/sendbird-uikit-react/assets/46333979/1c7f6ac1-a2f3-4d5d-87c9-0b4592509e88)
The structure has become much more stable, Thank you @bang9 !!